promrules.yaml

```diff
@@ -57,27 +57,70 @@ additionalPrometheusRulesMap:
                 HTTP probe returned error {{ $value }}
             expr: |
               probe_http_status_code <= 199 OR probe_http_status_code >= 400
-            for: 5m
+            for: 2m
             labels:
-              email_contact: global-it-operations@greenpeace.org
+              email_contact: pops@greenpeace.org
               gitlab_project: planet4-documentation-and-issues
-              severity: warning
+              severity: critical
               slack_channel: p4-slack-alert
           - alert: HttpSlowRequests
             annotations:
               description: '{{$labels.instance}} probe is taking longer than 2s'
               observability_url: >-
-                https://grafana.greenpeace.org/d/_nw9AwwMz/probe-monitoring-global?orgId=1&var-target={{$labels.instance}}
+                https://grafana.greenpeace.org/d/na0nQDXGk/planet4-end-point-monitoring?orgId=1&var-target={{$labels.instance}}
               runbook_url: >-
                 https://docs.google.com/document/d/1YgbkEDgENyxJ67QZ5VeVTh-uZfv8UCKvDGv553CvqBQ/edit#bookmark=id.89vqtjr3cb8p
               summary: '{{$labels.instance}} responding slowly'
               gitlab_incident_markdown: |
                  /label  ~\"Terraform Deployment\"\n HTTP probe slow
             expr: |
               avg_over_time(probe_http_duration_seconds[1m]) > 2
-            for: 5m
+            for: 2m
+            labels:
+              email_contact: pops@greenpeace.org
+              gitlab_project: planet4-documentation-and-issues
+              severity: medium
+              slack_channel: p4-slack-alert
+          - alert: PVCFullInFourDays
+            annotations:
+              description: >-
+                '{{ $labels.namespace }}/{{ $labels.persistentvolumeclaim }} is
+                expected to fill up within four days.   Currently {{ $value |
```

Comment:
The value here will be the amount of storage after x seconds in the predict linear. So probably not the value you are expecting